### ISSUE TITLE
Feature Req: []byte or interface{} etc for value

### ISSUE THREAD

**Issue by 5k3105** [couchbaselabs/vellum#15]
uint64 forces an extra lookup table for multiple valued terms ie word matching multiple meanings/ dictionary entries.

This was mentioned as an upcoming feature in a go conf video on youtube.

**Comment by 5k3105** [comment]
Anyway though, thanks for this lib! Working great on my dictionary of 2.1M unicode entries @ 6.3MB!

**Comment by mschoch** [comment]
Yes, thanks for opening up the issue.  We plan to get started on this feature shortly.
